x-pack/plugins/index_management/public/application/store/selectors/extension_service.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ExtensionsService } from '../../../services';
```

Comment:
@jloleysens  Is it recommended to import interfaces with `type` in front'?

Reply:
i think it would probably not make a difference

Reply:
@sebelga I don't thing it is necessary, but it is a TS feature that ensures at build time the file is not included, i.e., only used for dev. Since we were looking at bundle size here I thought we could add that guard here.

Reply:
Thanks @jloleysens I get it 👍